maestro-orchestra-models/src/main/java/maestro/orchestra/Commands.kt

```diff
@@ -856,10 +855,13 @@ data class StartRecordingCommand(
     }
 }
 
-data class AddMediaCommand(val mediaPaths: List<String>): Command {
+data class AddMediaCommand(
+    val mediaPaths: List<String>,
+    val label: String?
```

Comment:
Shouldn't this be having a default valu?

Reply:
To default it explicitly to null?

Reply:
Done